install-cluster/README.adoc

```diff
@@ -0,0 +1,88 @@
+= Install kubernetes cluster on AWS using kops
+:toc:
+
+This tutorial will walk you through how to install kubernetes cluster using kops.
+
+. Install kops and kubectl on your PC
+
+    brew update && install kops
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.goo\
+    gleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+
+. IAM user permission
+
+    Make sure the latest version of http://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS CLI]
+    is installed. User permissions being used must have these
+    http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies] attached
+    AmazonEC2FullAccess
+    AmazonRoute53FullAccess
+    AmazonS3FullAccess
+    IAMFullAccess
+    AmazonVPCFullAccess
+
+    Please review this link for additional info on IAM permission:
+    https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user
+
+. Create bucket to store k8s config
+
+    aws s3api create-bucket --bucket kubernetes-aws-config
+    # enable versioning and export
+    aws s3api put-bucket-versioning --bucket kubernetes-aws-config --versioning-configuration\
+    Status=Enabled
+    export KOPS_STATE_STORE=s3://kubernetes-aws-config
+
+. Create hosted zone on Route53
+
+    # If you own Route53 domain name, download https://github.com/stedolan/jq/wiki/Installation[jq]
+    # and run this command:
+    ID=$(uuidgen) && aws route53 create-hosted-zone --name kubernetes-aws.io \
+    --caller-reference $ID | jq .DelegationSet.NameServers
+    # See other means of creating DNS: https://github.com/kubernetes/kops/blob/master/docs/aws.md#configure-dns
+
+    # If you don't own domain name, you can create custom domain using Route53's private hosted zone
+    # you need to provide VPC info to run this command. See below how to create VPC (optional)
+    ID=$(uuidgen) && aws route53 create-hosted-zone --name k8s-aws.internal \
+    --vpc VPCRegion=us-east-1,VPCId=$VPCID --caller-reference $ID \
+    | jq .DelegationSet.NameServers
+
+. Create kubernetes cluster
+
+    kops create cluster --name cluster01.kubernetes-aws.io --state s3://kubernetes-aws-config
+    # review cluster info and make changes if necessary
+    kops get cluster
+    kops edit cluster cluster01.kubernetes-aws.io
+    # update and commit
```

Comment:
what edits are made to the cluster?

Reply:
I added (for ex, subnet sizing) into comments